src/components/resultBox/resultBox.css

```diff
@@ -54,7 +48,22 @@
     font-weight: 600;
   }
 
-  & .okButton {
-    margin-left: 20px;
+  & footer {
+    padding: 0px;
+    display: flex;
+    justify-content: space-between;
+    position: absolute;
+    bottom: 58px;
+
+    & > div > button {
+      width: 160px !important;
```

Comment:
Please double check if it is possible to avoid `!important`

Reply:
totally right, I just fixed